.github/workflows/automationTemplate.yaml

```diff
@@ -0,0 +1,146 @@
+name: Oryx-SDK-Runtime-Automation-Template
+
+on:
+  workflow_call:
+    inputs:
+      releaseDate:
+        required: false
+        type: string
+      platformName:
+        required: true
+        type: string
+      platformNameCamelCase:
+        required: true
+        type: string
+
+  workflow_dispatch:
+    inputs:
+      releaseDate:
+        description: 'The date to check for new releases. Format: yyyy-mm-dd'
+        required: false
+        type: string
+      platformName:
+        required: true
+        type: string
+        default: 'dotnet'
+        description: 'The platform name should be all lowercase'
+      platformNameCamelCase:
+        required: true
+        type: string
+        default: 'DotNetCore'
+        description: 'The value is used to generate the SDK & BaseImage pipeline name'
+
+env:
+  MAX_RETRIES: 10
+  TIMEOUT_SECONDS: 600
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Run SDK automation
+        run: |
+          releaseDate=${{ inputs.releaseDate }}
+          if ! [[ "${releaseDate}" =~ ^[0-9]{4}-[0-9]{2}-[0-9]{2}$ ]]
+          then
+            echo "release date is empty or invalid $releaseDate."
+            echo "Expecting date format yyyy-mm-dd"
+            releaseDate=`date +%F`
+            echo "Defaulting to todays date ${releaseDate}"
+          fi
+
+          echo "releaseDate=${releaseDate}" >> $GITHUB_ENV
+
+          dotnet run --project build/tools/Automation/Automation.csproj ${releaseDate} ${{ github.workspace }}
+
+      - name: Check if any git changes exist.
+        run: |
+          changes=$(git diff-index --name-only HEAD --)
+          echo ${changes}
+          if [ -z "${changes}" ]; then
+              echo "No git changes to commit. Cancelling workflow..."
+              exit 1
+          fi
+          git status
+
+      - name: Commit new ${{ inputs.platformNameCamelCase }} SDK versions
+        run: |
+          git status
+          git config user.name 'William Hernandez'
```

Comment:
Could you please clarify: do we need to set a user for this to work?
If yes, do we need to set a valid user, or can we set something more generic rather than a personal account?

Reply:
Double checking...

Reply:
A user needs to be specified. But I've updated to use Oryx Team instead.